st2actions/tests/unit/test_notifier.py

```diff
@@ -79,6 +79,5 @@ def test_notify_triggers(self):
         liveaction.start_timestamp = datetime.datetime.utcnow()
 
         dispatcher = NotifierTestCase.MockDispatcher(self)
-        notifier = Notifier(q_connection=None,
-                            trigger_dispatcher=dispatcher)
-        notifier.handle_action_complete(liveaction)
+        notifier = Notifier(None, [], trigger_dispatcher=dispatcher)
```

Comment:
Minor thing - please use keyword arguments where possible (`Notifier(connection=None, queues=[], ...`). It makes code easier to read and bugs easier to spot.